nicegui/testing/general_fixtures.py

```diff
@@ -0,0 +1,80 @@
+import importlib
+from typing import Generator, List, Type
+
+import pytest
+from starlette.routing import Route
+
+import nicegui.storage
+from nicegui import Client, app, binding, core, run, ui
+from nicegui.page import page
+
+# pylint: disable=redefined-outer-name
+
+
+def pytest_configure(config: pytest.Config) -> None:
+    """Add the "module_under_test" marker to the pytest configuration."""
+    config.addinivalue_line('markers',
+                            'module_under_test(module): specify the module under test which then gets automatically reloaded.')
```

Comment:
Why did you add `(module)`? What does it mean?

Reply:
It indicates that the marker needs a parameter. Like here: https://docs.pytest.org/en/stable/how-to/writing_plugins.html#registering-markers